engine/sbom/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Viewing the SBOM for Docker images
```

Comment:
Perhaps

```suggestion
title: View the SBOM for Docker images
```

Or should it be;

```suggestion
title: Generate the SBOM for Docker images
```

(should it be "image" instead of "Docker image"?)